flixel/system/FlxAssets.hx

```diff
@@ -152,7 +152,7 @@ class FlxAssets
 	 */
 	public static inline function getBitmapFromClass(source:Class<Dynamic>):BitmapData
 	{
-		return Type.createInstance(source, [0, 0]);
+		return Type.createInstance(source, []);
```

Comment:
@Gama11 I assume this isn't meant for normal bitmaps, but bitmap classes created using @:bitmap. we would never actually want to make a bitmap of 0x0, so this shouldn't be a breaking change, right?

Reply:
Are you sure this works on all targets? I thought this had to be here because width and height generally aren't optional parameters.

Reply:
BitmapData's width and height properties are not optional, but classes that use `@:bitmap` have all optional arguments. I tested EZPlatformer and Splitscreen on native mac, since they both use `FlxGraphic.fromClass(GraphicAuto)` and they had no issue